index.html

```diff
@@ -11,16 +11,25 @@
         <div class="navbar">
             <nav>
                 <a href="#">John Doe</a>
```

Comment:
- [ ] There is a huge gap consistently on the right side of the screen for all your website, please kindly fix that.

![Screenshot from 2022-07-08 23-31-44](https://user-images.githubusercontent.com/74501523/178073794-3ec53f5a-4be6-4431-9174-a126777792b1.png)